src/main/java/seedu/address/ui/meetup/MeetUpListPanel.java

```diff
@@ -72,8 +72,7 @@ private boolean doDateRangesOverlap(MeetUp meetUp1, MeetUp meetUp2) {
         LocalDateTime end2 = meetUp2.getTo().getDateTime();
 
         // Check if the date ranges overlap
-        return (start1.isBefore(end2) || start1.isEqual(end2))
-                && (end1.isAfter(start2) || end1.isEqual(start2));
+        return start1.isBefore(end2) && end1.isAfter(start2);
```

Comment:
Good implementation to ensure that we don't accept a start that is before an end timing.